modules/gapi/src/backends/common/serialization.cpp

```diff
@@ -734,6 +746,13 @@ I::OStream& ByteMemoryOutStream::operator<< (const std::string &str) {
     for (auto c : str) *this << c;
     return *this;
 }
+I::OStream& ByteMemoryOutStream::operator<< (const std::map<std::string, std::string> &map_str) {
+    *this << static_cast<uint32_t>(map_str.size());
+    for (const auto& strs : map_str) {
+        *this << strs.first << strs.second;
+    }
+    return *this;
+}
```

Comment:
Instead, I'd recommend to implement a generic version like this:  https://github.com/opencv/opencv/blob/4.4.0/modules/gapi/src/backends/common/serialization.hpp#L192

Reply:
Yeah, I saw this one. Added generic version